src/components/PathList.js

```diff
@@ -0,0 +1 @@
+export const API_DOMAIN = "http://127.0.0.1:8000";
```

Comment:
Domain does not include the `http` prefix
```suggestion
export const API_DOMAIN = "127.0.0.1:8000";
```

Reply:
Done